src/pages/EnspirePage/components/Register.js

```diff
@@ -7,7 +7,53 @@ class Register extends Component {
         //write your css in src/scss/apps/enspire/components/_register.scss
         return(
             <div className="w-100 h-100 enspire-register">
```

Comment:
```suggestion
            <div className="w-100 enspire-register">
```

Your content is going above screen height on small screens, so we don't need to enforce height: 100%